.agents/skills/shadcn-ui/vendor/shadcn/cli.md

```
@@ -2,9 +2,14 @@
 
 Configuration is read from `components.json`.
 
-> **IMPORTANT:** Always run commands using the project's package runner: `npx shadcn@latest`, `pnpm dlx shadcn@latest`, or `bunx --bun shadcn@latest`. Check `packageManager` from project context to choose the right one. Examples below use `npx shadcn@latest` but substitute the correct runner for the project.
+> **IMPORTANT:** Always run commands using the project's package runner: `npx shadcn@latest`,
+> `pnpm dlx shadcn@latest`, or `bunx --bun shadcn@latest`. Check `packageManager` from project
+> context to choose the right one. Examples below use `npx shadcn@latest` but substitute the correct
+> runner for the project.
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)